lib/tasks/admin.rake

```diff
@@ -107,7 +107,15 @@ namespace :admin do
     # Create jurisdiction for it does not already exist
     if jurisdiction == nil
       jurisdiction = Jurisdiction.create(name: jur_name , parent: parent)
-      unique_identifier = Digest::SHA256.hexdigest(jurisdiction.jurisdiction_path_string)
+
+      # create a 10 character, url-safe, base-64 string based on the SHA-256 hash of the jurisdiction path
+      unique_identifier = Base64::urlsafe_encode64([[Digest::SHA256.hexdigest(jurisdiction.jurisdiction_path_string)].pack('H*')].pack('m0'))[0, 10]
```

Comment:
So with the pack('m0') you're already getting a base 64 encoded string, is the call to `urlsafe_encode64` then just to make it url safe?

Reply:
correct, it's because pack('m0') returns `+` and `/` characters and calling `urlsafe_encode64` changes those to `-` and `_`